bridge/slack/handlers.go

```diff
@@ -228,33 +227,23 @@ func (b *Bslack) handleAttachments(ev *slack.MessageEvent, rmsg *config.Message)
 	}
 
 	// If we have files attached, download them (in memory) and put a pointer to it in msg.Extra.
-	for _, f := range ev.Files {
-		f := f
-		err := b.handleDownloadFile(rmsg, &f)
-		if err != nil {
+	for i := range ev.Files {
+		if err := b.handleDownloadFile(rmsg, &ev.Files[i]); err != nil {
 			b.Log.Errorf("Could not download incoming file: %#v", err)
 		}
 	}
 }
 
-var commentRE = regexp.MustCompile(`.*?commented: (.*)`)
```

Comment:
The regexp is no longer needed as an uploaded file with comment will be a simple Slack message with the file(s) attached in the `Files` field of the message and the comment in the `Text` field.